module/VuFind/src/VuFind/RecordTab/HierarchyTree.php

```diff
@@ -213,6 +208,16 @@ public function getSearchLimit()
         return $config->Hierarchy->treeSearchLimit ?? -1;
     }
 
+    /**
+     * Get the current active record. Returns record driver if found or null if ID invalid.
+     *
+     * @return ?\VuFind\RecordDriver\AbstractBase
+     */
+    public function getActiveRecord(): ?\VuFind\RecordDriver\AbstractBase
+    {
+        return null;
```

Comment:
Maybe worth adding a comment about why this always returns null, since the function-level comment suggests it's an error condition, but I assume it's happening for a reason in this context.

Reply:
Comment improved. It's overridden in CollectionHierarchyTree with actual functionality.